src/main/scala/com/yotpo/metorikku/code/steps/ToAvro.scala

```diff
@@ -0,0 +1,43 @@
+package com.yotpo.metorikku.code.steps
+
+import com.yotpo.metorikku.exceptions.MetorikkuException
+import org.apache.spark.sql.functions.struct
+import za.co.absa.abris.avro.functions.to_avro
+import za.co.absa.abris.avro.read.confluent.SchemaManager
+
+object ToAvro {
+
+  val message = "You need to send the following parameters to output to Avro format:" +
+    "table, schema.registry.url, schema.registry.topic, value.schema.name, value.schema.namespace." +
```

Comment:
also add this parameters:
kafka.url, schema.registry.topic

Reply:
schema.registry.topic already exists, adding kafka.url